frontends/web/src/routes/buy/info.tsx

```diff
@@ -45,7 +44,7 @@ export const BuyInfo = ({ code, accounts }: TProps) => {
       const options =
         supportedAccounts.map(({ name, code, coinCode }) => ({ label: `${name}`, value: code, coinCode, disabled: false }));
       setOptions(options);
-      getBalances(options);
+      setOptionBalances(options, setOptions);
```

Comment:
Yeah, you're definitely right. I updated the function to just take in input an options array and return an updated array with balances. I also removed the double call to the state setter in `buy/info.tsx` and `bitsurance/account.tsx`, as I don't see a big advantage in populating it twice just to avoid waiting for the balance calls to return.